packages/block/test/block.spec.ts

```diff
@@ -163,11 +163,14 @@ describe('[Block]: block functions', () => {
   }
 
   it('should test transaction validation - transaction not signed', async () => {
+    const common = new Common({ chain: Mainnet, hardfork: Hardfork.Osaka })
+    const maxTransactionGasLimit = paramsTx['7825'].maxTransactionGasLimit as number
+    // Create tx with gas limit over max (but not yet on Osaka)
     const tx = createLegacyTx({
-      gasLimit: 17000000,
+      gasLimit: maxTransactionGasLimit + 1,
       gasPrice: 7,
     })
```

Comment:
Nit: this test will fail once we default common to Osaka, would be better to explicitly mark this as pre-Osaka common here.